spec/components/provider_interface/key_dates_banner_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::KeyDatesBanner do
+  subject(:result) { render_inline(described_class.new) }
+
+  it 'renders the banner title' do
+    expect(result.text).to include('Important')
+  end
+
+  describe 'rendering the banner header' do
+    around do |example|
+      Timecop.freeze(time) { example.run }
+    end
+
+    context 'up to 20 days after the cycle opens' do
+      let(:time) { 10.business_days.after(CycleTimetable.apply_opens).end_of_day }
+
+      it 'does renders the non working period content' do
+        expect(result.text).not_to include("#{CycleTimetable.holidays[:christmas].first.to_s(:govuk_date)} to #{CycleTimetable.holidays[:christmas].last.to_s(:govuk_date)} do not count as working days")
+        expect(result.text).not_to include('The Christmas period will not be included when calculating dates for applications to be automatically rejected.')
```

Comment:
if we change the content these tests will fail, instead we can check that the text included matches the expected locales as so `t('key_dates_banner.title')` etc., what do you think?